study-builder/studies/pancan/emails/parental_consent_reminder.en.html

```diff
@@ -0,0 +1,20 @@
+#set( $text = {
+  "subject": "Reminder: Please sign the Count Me In Consent form",
+  "logo_alt": "Pancan Project logo",
+  "salutation_self": "Dear Participant,",
+  "salutation_parent": "Dear Parent,",
+  "thank_you": "Thank you for registering for Count Me In. We are now asking if you would be willing to sign our research consent form
+    to complete {{\#if proxy_firstName}}your child's{{else}}your{{/if}} enrollment and opt into different aspects of this study.",
+  "sign_consent": "Please Sign Consent Form",
+  "contact": "If you have questions at any point and would like to talk to a member of our study staff, please reach out to us at
+    <a href='mailto:${contact.email}' target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>${contact.email}</a>
+    or call <a href='tel:${contact.phone}' style='color: #7154ff; text-decoration: none; cursor: pointer;'>${contact.phone}</a>.",
```

Comment:
For email templates inline styles are important (some email clients can cut the external css).

We could split the string into several, but I don't think is is beneficial in the case. 
Because we have full template translations, not by strings.

Reply:
Ah I didn't know that, today I learned about importance of inline styles!